### PR TITLE
Add Hamta profile

### LINKED ISSUE #34: Add Hamta to the webpage

### PR DESCRIPTION
**Pull request by Hamtabardool**
# Fixes #34

### CODE REVIEW

File: 2-members.md [hunk 1]
```diff
@@ -69,8 +69,34 @@ banner_color: style2
...
+      <img src="assets/images/members/HamtaBardool.jpg" alt="" data-position="center center" />
+     </a>
+     <div class="content">
+      <div class="inner">
+       <header class="major">
+        <h3>Hamta Bardool</h3>
+       </header>
+       <li>Joined in Summer 2024.</li>
+       <li>Ph.D., Chemical Engineering, Purdue University/Shiraz University, 2023.</li>
+                   <p><b>Research topic</b>: Optimization,Process System Engineering, and Catalyst.</p>
```
Comment (bernalde) [inline 2-members.md:85]: I think we are not working on `catalyst `but on `catalysis`. Also, a space is missing after the comma between Optimization and Process. I would say out topic is: 
`Reactor and catalysis modeling, simulation, and optimization.`

File: 2-members.md [hunk 2]
```diff
@@ -69,8 +69,34 @@ banner_color: style2
...
+      <!-- <h4>Icons</h4> -->
+    <ul class="icons">
+     <li><a href="https://github.com/Hamtabardool" class="icon fa-github"><span class="label">Github</span></a></li>
+           <li><a href="www.linkedin.com/in/hamta-bardool-6ba418116/" class="icon fa-linkedin"><span class="label">Linkedin</span></a></li>
```
Comment (bernalde) [inline 2-members.md:89]: Make sure to add yourself to the SECQUOIA linkedin

**Review by bernalde — CHANGES_REQUESTED** [review]

File: 2-members.md (hunk 1 above)
Comment (Hamtabardool) [reply 2-members.md:85]: Sure, Prof. Bernal, 
I fixed it.

File: 2-members.md (hunk 2 above)
Comment (Hamtabardool) [reply 2-members.md:89]: Sure.

**Review by bernalde — APPROVED** [review]

**Pull request merged**